web/src/contexts/formContext.ts

```diff
@@ -49,7 +49,7 @@ export const createFormContext = <T>(): Context<FormContextType<T>> =>
   createContext<FormContextType<T>>({
     fieldStates: {} as ReducedFieldStates<keyof T>,
     runValidations: false,
-    reducer: () => ({} as ReducedFieldStates<keyof T>),
+    reducer: () => ({}) as ReducedFieldStates<keyof T>,
```

Comment:
This one seems out of place - is it changing the return type on this function? Why is this a trailing comma issue? It seems these parentheses are no longer doing any work here